cloudbuild.deploy.yaml

```diff
@@ -20,5 +20,5 @@ steps:
       - |
         set -e
         ./gcf/custom/gcp/deploy.sh datcom-stanford
-        ./gcf/custom/gcp/deploy.sh datcom-magiceye-dev
+        ./gcf/custom/gcp/deploy.sh datcom-mixer-encode
```

Comment:
For my own knowledge, settings here are not auto-deployed right? I think datcom-mixer-encode is actively used for climate trace instance?

Reply:
These will be auto-deployed. I think it's okay since this only affects GCF (not any user visible parts), and we almost always want to deploy latest to these instances.